src/main/kotlin/no/nav/syfo/behandlerdialog/kafka/KafkaUbesvartMelding.kt

```diff
@@ -78,10 +79,13 @@ class KafkaUbesvartMelding(private val database: DatabaseInterface) : KafkaConsu
         connection: Connection,
     ) {
         log.info("Received ubesvart melding with uuid: ${melding.referanseUuid}")
-        connection.createPersonOppgave(
-            melding = melding,
-            personOppgaveType = PersonOppgaveType.BEHANDLERDIALOG_MELDING_UBESVART,
-        )
+        val existingOppgave = connection.getPersonOppgaveByReferanseUuid(melding.referanseUuid)
+        if (existingOppgave == null) {
+            connection.createPersonOppgave(
+                melding = melding,
+                personOppgaveType = PersonOppgaveType.BEHANDLERDIALOG_MELDING_UBESVART,
+            )
+        }
```

Comment:
Hvis vi i fremtiden legger på flere personoppgaver per melding (feks avvist apprec), så vil denne logikken også hindre at man oppretter en purring. Men jeg tenker det gir mening fortsatt at påminnelse-oppgaven må "vike" for andre oppgaver.